runtime/crab/src/migration.rs

```diff
@@ -41,7 +41,16 @@ impl frame_support::traits::OnRuntimeUpgrade for CustomOnRuntimeUpgrade {
 }
 
 fn migrate() -> frame_support::weights::Weight {
-	frame_support::weights::Weight::zero()
+	// substrate
+	use frame_support::traits::StorageVersion;
+
+	// Some pallets ere added on chain after the migration.
+	// Thus, they never required the migration and we just missed to set the correct
+	// `StorageVersion`.
+	StorageVersion::new(4).put::<Scheduler>();
+	StorageVersion::new(1).put::<PolkadotXcm>();
```

Comment:
What's the meaning of this line? What would happen if we don't set it?

Reply:
https://github.com/itering/internal-security-audit/pull/53